### PR TITLE
Improving nodejs Docs

### LINKED ISSUE #402: Improve documentation
* Add more detailed documentation with punctuation.

### PR DESCRIPTION
**Pull request by Hribek25**
fixes #402

### CODE REVIEW

File: bindings/nodejs/examples/01_get_info.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+
+function run(){
+    const { ClientBuilder } = require('@iota/client');
+
+    // client will connect to testnet by default
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/01_get_info.js:5]: Maybe have at least one example with a custom node

File: bindings/nodejs/examples/09_transaction.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+    // client will connect to testnet by default
+    const client = new ClientBuilder().build();
+
+    const messageId = await client.messageSender()
+        .seed(IOTA_SEED_SECRET)
+        .output("atoi1qqydc70mpjdvl8l2wyseaseqwzhmedzzxrn4l9g2c8wdcsmhldz0ulwjxpz", 1000000)
+        .submit();
+
+    console.log(
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/09_transaction.js]: Formatting looks weird here

File: bindings/nodejs/examples/01_get_info.js (hunk 1 above)
Comment (Hribek25) [reply bindings/nodejs/examples/01_get_info.js:5]: Added an example with a custom node

File: bindings/nodejs/examples/09_transaction.js (hunk 2 above)
Comment (Hribek25) [reply bindings/nodejs/examples/09_transaction.js]: Slightly improved formatting

File: bindings/nodejs/examples/05b_get_output.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+async function run(){
+    const { ClientBuilder } = require('@iota/client');
+
+    // client will connect to testnet by default
+    const client = new ClientBuilder().build();
+
+    const output = await client.getOutput(a22cba0667c922cbb1f8bdcaf970b2a881ccd6e88e2fcce50374de2aac7c37720000);
+    console.log(
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/05b_get_output.js]: Sry, didn't notice it before, can you also format it here?

File: bindings/nodejs/examples/05c_find_outputs.js [hunk 4]
```diff
@@ -0,0 +1,13 @@
+async function run(){
+    const { ClientBuilder } = require('@iota/client');
+
+    // client will connect to testnet by default
+    const client = new ClientBuilder().build();
+
+    const outputs = await client.findOutputs(addresses=["atoi1qp9427varyc05py79ajku89xarfgkj74tpel5egr9y7xu3wpfc4lkpx0l86"]);
+    console.log(
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/05c_find_outputs.js]: Same here

File: bindings/nodejs/examples/06_simple_message.js [hunk 5]
```diff
@@ -0,0 +1,13 @@
+async function run(){
+    const { ClientBuilder } = require('@iota/client');
+
+    // client will connect to testnet by default
+    const client = new ClientBuilder().build();
+
+    const messageId = await client.postMessage();
+    console.log(
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/06_simple_message.js]: And here

File: bindings/nodejs/examples/05a_get_address_outputs.js [hunk 6]
```diff
@@ -0,0 +1,13 @@
+async function run(){
+    const { ClientBuilder } = require('@iota/client');
+
+    // client will connect to testnet by default
+    const client = new ClientBuilder().build();
+
+    const outputs = await client.getAddressOutputs('atoi1qp9427varyc05py79ajku89xarfgkj74tpel5egr9y7xu3wpfc4lkpx0l86');
+    console.log(
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/05a_get_address_outputs.js]: Here

File: bindings/nodejs/examples/04_get_balance.js [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+    console.log(
+        client.getAddressBalance("atoi1qp9427varyc05py79ajku89xarfgkj74tpel5egr9y7xu3wpfc4lkpx0l86")
+    );
+
+    const balance = await client.getBalance(IOTA_SEED_SECRET)
+        .accountIndex(0)
+        .initialAddressIndex(0)
+        .get();
+
+    console.log(
```
Comment (Thoralf-M) [inline bindings/nodejs/examples/04_get_balance.js]: Here

**Comment by Thoralf-M** [comment]
Sometimes you compared it to iota 1.0, do you really think that's useful? Maybe for some people that worked with the old client libraries before, but for most people it could maybe only be confusing/unnecessary information? I know it's done in different docs not only here, but maybe we should remove it from most docs and only have one `Changes from iota 1.0 to 1.5` section where that's explained
(Doesn't need to be decided for this PR)

**Review by Thoralf-M — APPROVED** [review]
Thank you!

**Comment by Hribek25** [comment]
> Sometimes you compared it to iota 1.0, do you really think that's useful? Maybe for some people that worked with the old client libraries before, but for most people it could maybe only be confusing/unnecessary information? I know it's done in different docs not only here, but maybe we should remove it from most docs and only have one `Changes from iota 1.0 to 1.5` section where that's explained
> (Doesn't need to be decided for this PR)

I think it is very good idea. Will discuss this within documentation team and probably will alter docs this way.

My initial assumption was that iota 1.0 users will be among trend setters even for Chrysalis wave that's why I wanted to somehow highlight breaking changes in comparsion to iota 1.0. But in overall, it is not needed for understanding chrysalis-based behavior for sure. You are right

**Pull request merged**